src/NServiceBus.Core/Logging/DefaultLoggerFactory.cs

```diff
@@ -46,7 +46,9 @@ public void Write(string name, LogLevel messageLevel, string message, Exception
             }
 
             var stringBuilder = new StringBuilder();
+#pragma warning disable PS0023 // Use DateTime.UtcNow or DateTimeOffset.UtcNow - For logging
             var datePart = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.fff");
```

Comment:
Would this work here?

```suggestion
            var datePart = DateTime.UtcNow.ToString("O");
```

E.g.

> 2021-07-14T07:46:23.4588246+00:00

Reply:
No, logging has always been in local time, so that when you're running locally you don't get confused. In production you'd run it on a server where UTC was the local time. In both cases you want the timestamp of the log message to correlate with real life.